test/e2e/components/vl-map-features-layer.js

```diff
@@ -0,0 +1,41 @@
+const VlMapLayer = require('./vl-map-layer');
+
+class VlMapFeaturesLayer extends VlMapLayer {
+  async getFeatures() {
+    return this.driver.executeScript(`return arguments[0].layer.getSource().getFeatures();`, this);
+  }
+
+  async getFeature(id) {
+    const feature = await this.driver.executeScript(`return arguments[0]._geoJSON.writeFeature(arguments[0].layer.getSource().getFeatures().find((feature) => feature.getId() === ${id}));`, this);
```

Comment:
getFeatureById bestaat ook

Reply:
Yep, had het gewoon overgeplaatst van vl-map-layer zonder er dieper naar te kijken, maar zal eens zien of we getFeatureById kunnen gebruiken hier.